Content.Shared/Lathe/LatheComponent.cs

```diff
@@ -22,6 +22,12 @@ public sealed class LatheComponent : Component
         [DataField("dynamicRecipes", customTypeSerializer: typeof(PrototypeIdListSerializer<LatheRecipePrototype>))]
         public readonly List<string> DynamicRecipes = new();
 
+        /// <summary>
+        /// All of the recipes that the lathe is capable to get using EMAG
+        /// </summary>
+        [DataField("emagRecipes", customTypeSerializer: typeof(PrototypeIdListSerializer<LatheRecipePrototype>))]
+        public readonly List<string> EmagRecipes = new();
```

Comment:
This should be a separate component, something like `EmagLatheRecipesComponent`.

Then you can define dynamic and static recipes on that component and just subscribe to `LatheGetRecipesEvent`. That way, you could have both static recipes and ones that only unlock when you also research the required technology.